packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -51,14 +51,15 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
 
             // Normalize relative import to absolute import
             if (importee.startsWith('.') && importer) {
+                const ext = path.extname(importer);
                 const normalizedPath = path.resolve(path.dirname(importer), importee);
-                const absPath = pluginUtils.addExtension(normalizedPath);
+                const absPath = pluginUtils.addExtension(normalizedPath, ext);
```

Comment:
this is assuming that the importer dictates the importee's extension. What if you're specifying the extension explicitly?

Reply:
yea, I guess I don't usually see imports specifying the extension (`import foo from ./foo.js`) so didn't think of that... I will update to check if importee has extension specified then if not assume it's the same as importer (since most .ts files will import other .ts files and most .js files will import other .js files).  Sound good?